drift/openapi/api.spec.yaml

```diff
@@ -34,3 +66,70 @@ components:
       properties:
         status:
           type: string
+    ComparisonReport:
+      required:
+        - facts
+        - header_mappings
+      properties:
+        facts:
+          type: array
+          items:
+            $ref: "#/components/schemas/FactComparison"
+        header_mappings:
```

Comment:
What is the idea behind `header_mappings` as the naming for this field? I don't find it obvious what it means as a data point compared to something like `systems`.

Reply:
I chose the name only because the data is likely to be used for the column headers in the frontend. I can change it to "systems" or something, that may make more sense

Reply:
Yea, I like to avoid if possible, API data that is "ui" specific in it's naming. When it's a public API, I think its better if it reflects what things actually are.